.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should add this file into gitignore.

Reply:
I will do it now. Tks!

Reply:
I tried adding .vscode to gitignore but it doesn't seem to work on my machine so I removed them manually. Can you review my changes? Tks

Reply:
The reason it won't work because you don't clear file cache yet. `git rm --cached filename`

Reply:
Oh tks! That one works. Sorry I'm inexperienced with this.

Reply:
Nothing. There are a lot of files. It will take me some time. When I finish reviewing it I will let you know. Thank.

Reply:
Tks. Take your time. Remind me if something does not work properly. I will try to resolve asap